.github/scripts/get_top_block.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+from sys import argv
+from substrateinterface import SubstrateInterface
+chain = SubstrateInterface(url=argv[1])
+number = chain.get_block()['header']['number']
```

Comment:
This approach got recommended to me by @DamianStraszak, since using `curl` is harder with `ws` and would require some `json` parsing (so likely also additional deps).